lib/components/Input/text-field-with-steppers/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { ComponentProps, forwardRef } from "react";
+import Input from "../base";
+import React from "react";
+import clsx from "clsx";
+import {
+    LabelPairedPlusSmRegularIcon,
+    LabelPairedMinusSmRegularIcon,
+} from "@deriv/quill-icons";
+import "./text-field-with-steppers.scss";
+import { TLeftOrRight } from "@types";
+
+export interface TextFieldWithSteppersProps extends ComponentProps<typeof Input> {
+    unit?: string;
+    decimals?: number;
+    unitPlacement?: TLeftOrRight;
+}
+
+export const TextFieldWithSteppers = forwardRef<HTMLInputElement, TextFieldWithSteppersProps>(
+    (props, ref) => {
+        const { value, onChange, disabled, textAlignment, placeholder = 0, unit = '', decimals = 2, unitPlacement = 'left' } = props;
+
+        const getFormatValue = (value: number) => {
+            let formatValue = value;
+
+            if (typeof formatValue === 'number') {
+                formatValue = parseFloat(formatValue.toFixed(decimals));
+            }
+
+            return formatValue;
+        }
+
+        const steppersSectionRight = (
+            <>
+                {textAlignment !== 'center' && (<button className={clsx("quill-input-steppers-decrement", disabled && 'quill-input-steppers--disabled')}
+                    onClick={() => {
+                        if (!disabled) {
+                            onChange?.({
+                                target: { value: getFormatValue(Number(value) - 1) },
+                            } as unknown as React.ChangeEvent<HTMLInputElement>);
+                        }
+                    }}>
+                    <LabelPairedMinusSmRegularIcon />
+                </button>)}
+                <button className={clsx("quill-input-steppers-increment", disabled && 'quill-input-steppers--disabled')}
+                    onClick={() => {
+                        if (!disabled) {
+                            onChange?.({
+                                target: { value: getFormatValue(Number(value) + 1) },
+                            } as unknown as React.ChangeEvent<HTMLInputElement>);
+                        }
+                    }}>
+                    <LabelPairedPlusSmRegularIcon />
+                </button>
+            </>
+        );
+
+        const steppersSectionLeft = (
+            <>
+                <button className={clsx("quill-input-steppers-decrement", disabled && 'quill-input-steppers--disabled')}
```

Comment:
This code looks like first part of `steppersSectionRight`. is it possible to extract them and reuse?

Reply:
There's no point or benefit of reusing this.